cdc/scheduler/internal/v3/replication/replication_manager.go

```diff
@@ -499,9 +513,33 @@ func (r *Manager) AdvanceCheckpoint(
 	if slowestTableID != 0 {
 		r.slowestTableID = slowestTableID
 	}
+
+	r.logSlowTableInfo(newCheckpointTs, currentTime)
+
 	return newCheckpointTs, newResolvedTs
 }
 
+func (r *Manager) logSlowTableInfo(checkpointTs model.Ts, currentTime time.Time) {
+	checkpointLag := currentTime.Sub(oracle.GetTimeFromTS(checkpointTs))
+	if checkpointLag < logSlowTablesLagThreshold ||
+		time.Since(r.lastLogSlowTablesTime) < logSlowTablesInterval {
+		return
+	}
+	for _, table := range r.slowTableHeap {
```

Comment:
How about just logging the slowest table? So we don't need a heap.

Reply:
May be the slowest table is not enought for us to locate the problem if there are more than one slow table in a changefeed. What’s your opinion on this matter? @nongfushanquan

Reply:
We only keep the Min-K tables in the heap, I think the cost is acceptable.

Reply:
Only the slowest table affects the changefeed lag, so I think it's enough for the most cases.

Reply:
I think "min-k" tables confuse us if there is only one slow table, which one should we looking for?

Reply:
After https://github.com/pingcap/tiflow/pull/7408 merged, other tables' progress is not hindered by the slowest table anymore. Let say there are 3 tables lagging, `t1` lags 10 minutes,  `t2` lags 8 minutes, `t3` lags 7 minutes, we should look for them all. This log is helpful in such condiction.